data/product.d/fedora-iot.conf

```diff
@@ -0,0 +1,11 @@
+# Anaconda configuration file for Fedora IoT.
```

Comment:
Could you please extend the method `fedora_products_test`, so the config file can be validated by our unit tests?

Reply:
Done.